src/04_printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print("x is %d, y is %g, z is %s" % (x, round(y, 2), z))
```

Comment:
if you wanted to include the quotations in "I like turtles!" in this line you could put backslashes around %s. 

`print("x is %d, y is %g, z is \"%s\"" % (x round(y, 2), z))`